nodebuilder/share/share.go

```diff
@@ -87,3 +86,30 @@ func (api *API) GetSharesByNamespace(
 ) ([]share.Share, error) {
 	return api.Internal.GetSharesByNamespace(ctx, root, namespace)
 }
+
+type module struct {
+	shareService *service.ShareService
+}
```

Comment:
optional nit: we can embed it and only override `SharesAvailable`, but that's very optional

Reply:
already pushed when saw your comment 😄